app/views/admin/world_location_news/show.html.erb

```diff
@@ -1,24 +1,25 @@
-<% page_title @world_location_news.name %>
-<% page_class "world-location-show" %>
-
-<%= content_tag_for(:div, @world_location_news) do %>
-  <div class="world-location-header">
-    <h1>
-      <span class="name"><%= @world_location_news.name %></span>
-    </h1>
-    <p>
-    <%= link_to "View on website", @world_location_news.public_url(cachebust_url_options) %>
-  </div>
-
-  <section class="world-location-details">
-    <%= tab_navigation_for(@world_location_news) %>
-
-    <div class="mission-statement lead">
+<% content_for :page_title, @world_location_news.name %>
+<% content_for :context, "World location" %>
+<% content_for :title, @world_location_news.name %>
+<% content_for :title_margin_bottom, 4 %>
+<%= link_to "View on website", @world_location_news.public_url(cachebust_url_options), class: "govuk-link
+govuk-!-margin-bottom-8" %>
+<div class="govuk-!-margin-bottom-8">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "components/secondary_navigation", {
+        aria_label: "Document tabs",
+        items: secondary_navigation_tabs_items(current_user, @world_location_news, request.path)
+      } %>
+      <div class="govuk-!-margin-bottom-4">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Edit",
+          secondary_solid: true,
+          href: link_to("Edit", [:edit, :admin, @world_location_news])
```

Comment:
This is inserting a link within a link.

<img width="671" alt="Screenshot 2023-03-21 at 10 52 58 am" src="https://user-images.githubusercontent.com/4599889/226585520-8cddd6b3-ce24-436d-bec5-4fa000634afd.png">